[2022-03-18 04:46:35,979] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: pbp_bucket_ingestion_dag.get_pbp_data scheduled__2014-12-06T04:00:00+00:00 [queued]>
[2022-03-18 04:46:36,016] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: pbp_bucket_ingestion_dag.get_pbp_data scheduled__2014-12-06T04:00:00+00:00 [queued]>
[2022-03-18 04:46:36,018] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-03-18 04:46:36,022] {taskinstance.py:1239} INFO - Starting attempt 2 of 2
[2022-03-18 04:46:36,029] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-03-18 04:46:36,065] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): get_pbp_data> on 2014-12-06 04:00:00+00:00
[2022-03-18 04:46:36,077] {standard_task_runner.py:52} INFO - Started process 611 to run task
[2022-03-18 04:46:36,091] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'pbp_bucket_ingestion_dag', 'get_pbp_data', 'scheduled__2014-12-06T04:00:00+00:00', '--job-id', '6299', '--raw', '--subdir', 'DAGS_FOLDER/pbp_ingestion_dag.py', '--cfg-path', '/tmp/tmpbyrlzcmr', '--error-file', '/tmp/tmpau99v52_']
[2022-03-18 04:46:36,099] {standard_task_runner.py:77} INFO - Job 6299: Subtask get_pbp_data
[2022-03-18 04:46:36,269] {logging_mixin.py:109} INFO - Running <TaskInstance: pbp_bucket_ingestion_dag.get_pbp_data scheduled__2014-12-06T04:00:00+00:00 [running]> on host a2ba0547c319
[2022-03-18 04:46:36,403] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=pbp_bucket_ingestion_dag
AIRFLOW_CTX_TASK_ID=get_pbp_data
AIRFLOW_CTX_EXECUTION_DATE=2014-12-06T04:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2014-12-06T04:00:00+00:00
[2022-03-18 04:46:36,671] {python.py:175} INFO - Done. Returned value was: None
[2022-03-18 04:46:36,716] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=pbp_bucket_ingestion_dag, task_id=get_pbp_data, execution_date=20141206T040000, start_date=20220318T044635, end_date=20220318T044636
[2022-03-18 04:46:36,815] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-18 04:46:36,922] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
